pkg/processor/k8sprocessor/factory.go

```diff
@@ -49,6 +49,7 @@ func createDefaultConfig() config.Processor {
 	return &Config{
 		ProcessorSettings: config.NewProcessorSettings(config.NewID(typeStr)),
 		APIConfig:         k8sconfig.APIConfig{AuthType: k8sconfig.AuthTypeServiceAccount},
+		Extract:           ExtractConfig{Delimiter: ", "},
```

Comment:
Can we name the default and use it (the defined `const`)?

Reply:
fixed